### PR TITLE
Switch away from travis

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
TODO:
 - testing
   - `ember new` in a new repo, pushed to Github
      - [ ] with npm
      - [ ] with yarn
   - `ember addon` in a new repo, pushed to Github
     - [ ] with npm
     - [ ] with yarn
  - [ ] replace automated ember-cli tests that check for a .travis.yml file for checking for gh actions files
     

For Both:
 - split linting and tests in to separate jobs to maximize parallel running to try to best decrease over all CI time
 - support `[skip ci]` magic commit messages
 - use Volta
 
For Apps:
 - build once for tests and production
 - cache dependencies between jobs, because GH Action's cache is faster than re-installing (even for small monorepos)
   - it's always easier to remove the cache and switch to install on every job that to figure out how to get the cache working
   - this makes it easier for folks to scale and add things like netlify deploy and deploy preview without relying on external deploy infrastructure (which can have predatory policy changes / send you bills when you've been on the free tier) -- example deploy config (not included in this PR): https://github.com/NullVoxPopuli/limber/blob/main/.github/workflows/ci.yml#L190-L276
 
 For Addons:
  - run on a weekly cron so that maintained projects can keep an eye on what things break over time (if any)

### CODE REVIEW

File: blueprints/addon/files/.github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  tests:
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+    name: Base Tests
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node:
+          - "12"
+          - "14"
```
Comment (snewcomer) [inline blueprints/addon/files/.github/workflows/lint.yml:20]: should we just chop this down to node 14.  Or even 14 and 16?

File: blueprints/addon/files/.github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+name: "Lint"
```
Comment (snewcomer) [inline blueprints/addon/files/.github/workflows/ci.yml:1]: I'm guessing this is the lint.yml and the other is the ci.yml?

**Review by snewcomer — COMMENTED** [review]
This is great!  

One way I see this blueprint also being useful is a more basic template (one file) with documentation on additional configuration options (e.g. ember-cli-update).  I guess it is a question of including everything out of the box or simplicity with extensive documentation.

File: blueprints/addon/files/.github/workflows/lint.yml (hunk 1 above)
Comment (NullVoxPopuli) [reply blueprints/addon/files/.github/workflows/lint.yml:20]: good callout!

**Pull request closed without merging**